server/resources/translations/common.en_fr_bilingual.json

```diff
@@ -8,6 +8,9 @@
   "buttonLowBatteryInitial": "La batterie pour le Button de la {{ buttonDisplayName }} est basse, et a besoin d'être rechargée. Pour voir un vidéo montrant comment recharger la batterie, allez au https://www.youtube.com/watch?v=h7_5ff-_r5A\n---\nThe battery for the {{ buttonDisplayName }} Button is low, and needs recharging.\n\nTo watch a video showing how to recharge the battery, go to https://www.youtube.com/watch?v=h7_5ff-_r5A",
   "buttonLowBatteryReminder": "La batterie pour le Button de la {{ buttonDisplayName }} est toujours basse, et a besoin d'être rechargée. Pour voir un vidéo montrant comment recharger la batterie, allez au https://www.youtube.com/watch?v=h7_5ff-_r5A\n---\nThe battery for the {{ buttonDisplayName }} Button is still low, and needs recharging.\n\nTo watch a video showing how to recharge the battery, go to https://www.youtube.com/watch?v=h7_5ff-_r5A",
   "buttonLowBatteryNoLonger": "La batterie pour le Button de la {{ buttonDisplayName }} n'est plus basse. Merci.\n---\nThe battery for the {{ buttonDisplayName }} Button is no longer low. Thank you.",
+  "buttonStatusChangeDisconnected": "Les boutons suivants ont été déconnectés: {{ buttonDisplayNames }}.",
+  "buttonStatusChangeReconnected": "Les boutons suivants ont été reconnectés: {{ buttonDisplayNames }}.",
+  "buttonStatusChangeStart": "Des modifications de connexion ont été apportées aux boutons de {{ clientDisplayName }}.",
```

Comment:
These messages should be in both English and French because this client wants bilingual messages. Could you please add `\n---\n` followed by the English text after each of these translations?

We have also been using "Buttons" with a capital "B" even in our French text because it's more like a brand name than an object. So just replace `boutons` with `Buttons` in these three strings.

If you haven't already, please ask Oona to get these translated. If she seems to take too long, you can get the contact information for Janick at CIPTO who did the other ones and ask them about it.

Reply:
Reminder to myself to ask Oona for translations of these messages.